scenes/menus/title/components/main_menu.gd

```diff
@@ -0,0 +1,28 @@
+extends HBoxContainer
+
+signal start_pressed
+signal credits_pressed
+
+@onready var button_box: VBoxContainer = %ButtonBox
+@onready var start_button: Button = %StartButton
+
+
+func _ready() -> void:
+	_on_visibility_changed()
+
+
+func _on_start_button_pressed() -> void:
+	start_pressed.emit()
+
+
+func _on_credits_button_pressed() -> void:
+	credits_pressed.emit()
+
+
+func _on_quit_button_pressed() -> void:
+	get_tree().quit()
```

Comment:
Of course this doesn't work on the web platform! We should hide the quit button if running on the web.

Reply:
Alternatively it's possible to handle the quit request.

Reply:
Pushed a patch to hide the quit button on the web.